ts/Core/Axis/Axis.ts

```diff
@@ -1833,12 +1833,18 @@ class Axis {
         // This is in turn needed in order to find tick positions in ordinal
         // axes.
         if (isXAxis && !secondPass) {
+            // First process all series assigned to that axis.
             axis.series.forEach(function (series): void {
+                // Allows filtering out points outside the plot area.
+                series.forceCrop = series.forceCropping();
```

Comment:
`forceCropping` is defined only in Highstock, so this will break all Highcharts charts: https://jsfiddle.net/BlackLabel/3p7ftcxh/

Replace with simple `series.forceCrop = series.forceCropping && series.forceCropping();`

PS: Why didn't TS warn about the issue?

Reply:
😮  This is the danger of always loading all the modules in the karma tests...